letsdrill.rb

```diff
@@ -1,24 +1,28 @@
-def get_letter_grade(integer)
- 
-  #Put your code here!
 
+def get_letter_grade(number_grade)
+  case number_grade
+    when 90..100 then 'A'
+    when 80...90 then 'B'
+    when 70...80 then 'C'
+    when 60...70 then 'D'
+    else 'F'
+  end
 end
 
-def shortest_string(array)
-
-  #Put your code here!
-
+def shortest_string(words)
+  unless words.length == 0
+    words.min_by &:length
+  end
```

Comment:
You can remove the `unless` clause around `words.min_by`. Since `words.min_by &:length` will return nil if the array is empty, you will get the same result.